codegen/js-gen/src/main/scala/io/kalix/codegen/js/EntityServiceSourceGenerator.scala

```diff
@@ -99,7 +99,7 @@ object EntityServiceSourceGenerator {
     val typedefPath =
       sourceDirectory.toAbsolutePath
         .relativize(generatedSourceDirectory.toAbsolutePath)
-        .resolve(service.fqn.name.toLowerCase())
+        .resolve(entity.fqn.name.toLowerCase())
```

Comment:
small fix, the generated files use the entity name not the service name, typedef imports were wrong